query-builder-elasticsearch.js

```diff
@@ -108,6 +110,8 @@
                         // we express this with a sub boolean query and must_not.
                         if (data.condition === 'OR' && (rule.operator === 'not_equal' || rule.operator === 'not_in')) {
                             return {'bool': {'must_not': [part]}}
+                        } else if (rule.operator === 'is_null' || rule.operator === 'is_not_null') {
```

Comment:
I think that this causes a bug when you do use a OR condition with one of the branches that uses the is_null (must not exist) operator. I made a test to show this.

```    
QUnit.test("OR condition and null op", function (assert) {

        $b.queryBuilder({
            filters: basic_filters,
            rules: {
                condition: 'OR',
                rules: [
                    {id: 'name', field: 'name', operator: 'is_not_null'},
                    {id: 'price', field: 'price', operator: 'is_null'}
                ]
            }
        });

        assert.deepEqual(
            $b.queryBuilder('getESBool'),
            {"bool": {"should":[
                {"exists":{"field":"name"}},
                {"bool":{"must_not": {"exists": {"field":"price"}}}}
            ]}},
            'Should build a should query with a bool subquery'
        );

    });
```